0001-Add-unit-test-to-retrieve-tuple-across-assembly-boun.patch

```diff
@@ -0,0 +1,351 @@
+From e430d95c96041334f7c17c64487d9b50697210bc Mon Sep 17 00:00:00 2001
```

Comment:
I don't think that this patch file should be commited to the repo...
I assume it was added by accident.

Reply:
Well spotted